src/core/network/workspacemanager.cpp

```diff
@@ -169,22 +169,22 @@ void WorkspaceManager::load(std::istream& stream, const std::string& refPath,
     deserializers_.invoke(deserializer, exceptionHandler);
 }
 
-void WorkspaceManager::save(const std::string& path, const ExceptionHandler& exceptionHandler,
+void WorkspaceManager::save(std::string_view path, const ExceptionHandler& exceptionHandler,
                             WorkspaceSaveMode mode) {
-    auto ostream = filesystem::ofstream(path);
+    auto ostream = filesystem::ofstream(std::string(path));
```

Comment:
maybe we should provide an override for string_view for both ofstream and ifstream

Reply:
problem it that ifstream needs a string or ah fs::path. hence Im sticking with string until we can switch to fs::path